library/src/main/java/net/vrgsoft/library/LinkCrawler.kt

```diff
@@ -125,90 +126,87 @@ class LinkCrawler {
                     || lowerCase.contains("property='og:url'")
                     || lowerCase.contains("name=\"url\"")
                     || lowerCase.contains("name='url'"))
-                updateMetaTag(metaTags, "url", separeMetaTagsContent(match))
+                updateMetaTag(metaTags, "url", separateMetaTagsContent(match))
             else if (lowerCase.contains("property=\"og:title\"")
                     || lowerCase.contains("property='og:title'")
                     || lowerCase.contains("name=\"title\"")
                     || lowerCase.contains("name='title'"))
-                updateMetaTag(metaTags, "title", separeMetaTagsContent(match))
+                updateMetaTag(metaTags, "title", separateMetaTagsContent(match))
             else if (lowerCase
                     .contains("property=\"og:description\"")
                     || lowerCase
                     .contains("property='og:description'")
                     || lowerCase.contains("name=\"description\"")
                     || lowerCase.contains("name='description'"))
-                updateMetaTag(metaTags, "description", separeMetaTagsContent(match))
+                updateMetaTag(metaTags, "description", separateMetaTagsContent(match))
             else if (lowerCase.contains("property=\"og:image\"")
                     || lowerCase.contains("property='og:image'")
                     || lowerCase.contains("name=\"image\"")
                     || lowerCase.contains("name='image'"))
-                updateMetaTag(metaTags, "image", separeMetaTagsContent(match))
+                updateMetaTag(metaTags, "image", separateMetaTagsContent(match))
         }
 
         return metaTags
     }
 
     private fun updateMetaTag(metaTags: MutableMap<String, String>, url: String, value: String?) {
-        if (value != null && value.isNotEmpty()) {
-            metaTags.put(url, value)
+        if (value?.isNotEmpty() == true) {
```

Comment:
Simplify this with value?.isNotEmpty()

Reply:
@pawlo2102 it won't work like that unfortunately

Reply:
The return type would be `Boolean?` whereas if expected `Boolean`